lib/modules/apostrophe-workflow-pages/index.js

```diff
@@ -222,9 +222,31 @@ module.exports = {
           req.locale = locale;
 
           return async.eachSeries(parked, function(item, callback) {
+            convertToLocalizedSlug(item, locale);
             return self.implementParkOne(req, item, callback);
           }, callback);
 
+          function convertToLocalizedSlug (item, locale) {
+            if (locale) {
+              var liveLocale = workflow.liveify(locale);
+              if (typeof item.slug === 'object') {
+                if (!item.slug[liveLocale] && !item.slug._default) {
+                  throw new Error('apostrophe-workflow: mising default value in localized parked page');
```

Comment:
Make this error message say _default so they don't add default and wonder why it does not work

Reply:
Ok